components/schemes/Color/src/MDCLegacyColorScheme.m

```diff
@@ -82,7 +82,7 @@ - (UIColor *)lighterColorForColor:(UIColor *)color {
   if ([color getHue:&hue saturation:&saturation brightness:&brightness alpha:&alpha]) {
     return [UIColor colorWithHue:hue
                       saturation:saturation
-                      brightness:(CGFloat)fminf((float)brightness + 0.2f, 1)
+                      brightness:(CGFloat)fmin(brightness + 0.2, 1)
```

Comment:
no need for casting here and below?

Reply:
I was on the fence about this. It seemed silly to take a CGFloat and force it to a float and then assign the output back to a CGFloat. I switched to `fmin` to keep things in double precision all the way through.

Reply:
I went back on my decision and reverted the changes. I think it's better to put refactorings (even small ones) into follow-up work.